src/assets/scss/introjs.scss

```diff
@@ -0,0 +1,103 @@
+.introjs-tooltip {
+    background-color: white;
+    border-radius: 9px;
+}
+.introjs-helperNumberLayer{
+    display: none;
+}
+.introjs-helperLayer{
+    border-radius: 9px;
+}
+.introjs-donebutton{
+    color: white !important;
+    background-image: none !important;
+    background-color: #508197 !important;
+    text-shadow: none;
+    border: none;
+    border-radius: 3px;
+    float: right !important;
+    font-family: "ProximaNova";
```

Comment:
You should be able to apply this font-family to a parent css class so you don't have to apply it in all of the below classes as well